src/content/releases/_index.en.md

```diff
@@ -12,7 +12,7 @@ weight = 15
 
 * Lighthouse no longer depends KubeFed. All metadata exchange is handled over the Broker as MultiClusterService CRs.
 * Experimental **Wireguard** support has been added as a pluggable CableDriver option in addition to the current default IPsec.
-* Submariner reports the active and passive gateways as a gateway.submariner.io CRD.
+* Submariner reports the active and passive gateways as a gateway.submariner.io CR.
```

Comment:
```suggestion
* Submariner reports the active and passive gateways as a gateway.submariner.io resource.
```

I prefer a meaningful word rather than an acronym that a reader might not be familiar with.

Reply:
or better 'custom resource'

Reply:
yeah we could add "custom" but I don't think that adds any value. I think it's implied that it's custom but, regardless, I don't think it's even relevant in documentation.

Reply:
Actually Ii'm  more referring to other parts of the docs, eg where we refer to Multiclusterservice. I suggested we refer to it as a _resource_  rather than via acronym or _custom resource_ which is think is too much since we refer to it a lot. Here it doesn't really matter if we call it _resource_ or _custom resource_.

Reply:
:+1: